snippets/csharp/api/system/datetimeoffset/parseexact/parseexact-iso8601-2.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Globalization;
+
+public class Example
+{
+   public static void Main()
+   {
+      string[] dateStrings = { "2018-08-18T12:45:16.0000000Z",
+                               "2018/08/18T12:45:16.0000000Z",
+                               "2018-18-08T12:45:16.0000000Z",
+                               "2018-08-18T12:45:16.0000000",                               
+                               " 2018-08-18T12:45:16.0000000Z ",
+                               "2018-08-18T12:45:16.0000000+02:00",
+                               "2018-08-18T12:45:16.0000000-07:00" }; 
+      
+      ParseWithISO8601(dateStrings, DateTimeStyles.None);
+      Console.WriteLine("\n-----\n");
+      ParseWithISO8601(dateStrings, DateTimeStyles.AllowWhiteSpaces);
+      Console.WriteLine("\n-----\n");
+      ParseWithISO8601(dateStrings, DateTimeStyles.AdjustToUniversal);
+      Console.WriteLine("\n-----\n");
+      ParseWithISO8601(dateStrings, DateTimeStyles.AssumeLocal);
+      Console.WriteLine("\n-----\n");
+      ParseWithISO8601(dateStrings, DateTimeStyles.AssumeUniversal);   }
+
+   private static void ParseWithISO8601(string[] dateStrings, DateTimeStyles styles)
+   {   
+      Console.WriteLine($"Parsing with {styles}:");
+      foreach (var dateString in dateStrings)
+      {
+         try {
+            var date = DateTimeOffset.ParseExact(dateString, "O", null, styles);
+            Console.WriteLine($"   {dateString} --> {date:yyyy-MM-dd HH:mm:ss.FF zzz}");
```

Comment:
really minor nit: Do you want to use a field width specifier on `dateString` to get them all to line up: `{dataString,-36}`

I think a field with of 36 works for every snippet in these examples.